app/src/main/java/com/sunasterisk/rememory/ui/addwork/AddWorkFragment.kt

```diff
@@ -0,0 +1,32 @@
+package com.sunasterisk.rememory.ui.addwork
+
+import android.annotation.SuppressLint
+import android.widget.PopupMenu
+import com.sunasterisk.rememory.BaseFragment
+import com.sunasterisk.rememory.R
+import kotlinx.android.synthetic.main.fragment_add_work.*
+
+class AddWorkFragment : BaseFragment(), AddWorkContract.View {
+    override val layoutResource get() = R.layout.fragment_add_work
+    override fun initComponent() {
+        classify()
+    }
+
+    @SuppressLint("SetTextI18n")
+    private fun classify() {
+        btnClassify.setOnClickListener {
+            val classifyMenu = PopupMenu(activity, btnClassify)
+            classifyMenu.menuInflater.inflate(R.menu.classify_menu, classifyMenu.menu)
+            classifyMenu.setOnMenuItemClickListener { item ->
+                when (item.itemId) {
+                    R.id.classifyWork -> btnClassify.text = "Work"
+                    R.id.classifyEntertainment -> btnClassify.text = "Entertainment"
+                    R.id.classifyFamily -> btnClassify.text = "Family"
+                    R.id.classifyMyself -> btnClassify.text = "Myself"
+                }
+                true
+            }
+            classifyMenu.show()
```

Comment:
phần này tách ra 1 hàm riêng nhé, showClassifyMenu() 
```suggestion
        private fun showClassifyMenu(){
            PopupMenu(activity, btnClassify).apply {
                .menuInflater.inflate(R.menu.classify_menu, classifyMenu.menu)
                .setOnMenuItemClickListener { item ->
                    when (item.itemId) {
                       R.id.classifyWork -> btnClassify.text = getString(R.string.title_work)
                       R.id.classifyEntertainment -> btnClassify.text = <Don't put hard code here>
                       R.id.classifyFamily -> btnClassify.text = <Don't put hard code here>
                       R.id.classifyMyself -> btnClassify.text = <Don't put hard code here>
                   }
                true
               }
                show()
            }
        }
            
```